src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -5,6 +5,7 @@
     <DefineConstants>$(DefineConstants);HTTP_DLL</DefineConstants>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Linux;$(NetCoreAppCurrent)-OSX;$(NetCoreAppCurrent)-FreeBSD;$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
     <Nullable>enable</Nullable>
+    <DefineConstants>FEATURE_QUIC_PUBLIC</DefineConstants>
```

Comment:
What is this for?

Reply:
a remnant of very early code where the classes were still internal. I will remove it.